blyp/Views/ContentView.swift

```diff
@@ -18,6 +18,7 @@ struct ContentView: View {
             if user.loginState == .loggedIn {
                 MainView()
             } else if user.loginState == .signingUp {
+                OnboardingView()
                 SignUpView()
             } else if user.loginState == .loggedOut {
                 LoginView()
```

Comment:
Were we going to put the OnboardingView in the  LoginView or is it supposed to be there? I can't remember what the designers decided lol

Reply:
I thought that it was going to be shown before someone signs up, but I could be wrong llolol